vllm/model_executor/layers/quantization/modelopt.py

```diff
@@ -1125,8 +1343,11 @@ def apply(
                 expert_map=expert_map)
 
         if self.fused_experts is None:
-            # If no modular kernel is provided, use cutlass_moe_fp4 for TP case
-            # only (no EP).
+            # If no modular kernel is provided, use cutlass_moe_fp4 or
+            # flashinfer low latency kernel for TP case only (no EP).
```

Comment:
I'm not seeing how this is using the flashinfer low latency kernel? I would think this assert should be before calling flashinfer_fp4_cutlass_moe_forward

Reply:
Right, the assert is not correctly positioned. Fixed.